pkg/platform/model/inventory.go

```diff
@@ -51,17 +51,19 @@ func IngredientByNameAndVersion(language, name, version string, prefix Namespace
 	}
 
 	for _, ingredient := range results {
-		if ingredient.Ingredient.Name == nil || *ingredient.Ingredient.Name != name {
-			continue
-		}
+		for _, feature := range ingredient.LatestVersion.ProvidedFeatures {
```

Comment:
Can we assume that the provided features are the same for all versions of an ingredient?  Or do we have to iterate over all versions and features?

Reply:
`provided_features` is only available via `latest_version` in the search response. `latest_version` should represent the full ingredient version data model as noted in the response here: https://platform.activestate.com/sv/inventory-api-v1/docs/#operation/searchIngredients